lms/djangoapps/courseware/views.py

```diff
@@ -1911,3 +1910,25 @@ def financial_assistance_form(request):
             }
         ],
     })
+
+
+def is_sneakpeek_allowed(user, course, course_key):
```

Comment:
This isn't a dealbreaker for now since we're trying to get this out,
but moving forward, new functions like this should be moved to a new file.
This prevents conflicts, should edX add anything to the bottom of this file.